### ISSUE TITLE
Zoological autonym missing authorship

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#640]
**Subspecies missing authorship**

I think this should show as something like:
_Chilomycterus spinosus spinosus_ (Linnaeus,1758)

[Submitted on behalf of user over email]

cc @mdoering

-----
fbitem-species7193801
User provided contact info: timrobertson100
System: Chrome 61.0.3163 / Mac OS X 10.11.5
Referer: https://www.gbif.org/species/7193801
Window size: width 1690 - height 939
[API log](http://elk.gbif.org:5601/app/kibana#/discover/UAT-Varnish-403s?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2017-11-07T08:43:49.630Z',mode:absolute,to:'2017-11-07T08:49:49.630Z'))&_a=(columns:!(request,response,clientip),filters:!(),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E499%20AND%20(request:%22%2F%2Fapi.gbif.org%22)')),sort:!('@timestamp',desc))&indexPattern=uat-varnish-*&type=histogram)
[Site log](http://elk.gbif.org:5601/app/kibana#/discover/UAT-Varnish-403s?_g=(refreshInterval:(display:Off,pause:!f,value:0),time:(from:'2017-11-07T08:43:49.630Z',mode:absolute,to:'2017-11-07T08:49:49.630Z'))&_a=(columns:!(request,response,clientip),filters:!(),index:'prod-varnish-*',interval:auto,query:(query_string:(analyze_wildcard:!t,query:'response:%3E399%20AND%20(request:%22%2F%2Fdemo.gbif.org%22)')),sort:!('@timestamp',desc))&indexPattern=uat-varnish-*&type=histogram)

**Comment by mdoering** [comment]
This is the right way to deal with autonyms in botany - they do not have an authorship.
But apparently they should use the species author in zoology: http://www.nhm.ac.uk/hosted-sites/iczn/code/index.jsp?nfv=true&article=47

**Comment by andzandz11** [comment]
Well, no and yes.
The Vienna code forbids a NEW author citation, if an autonym is automatically created. That does not mean that the original citation disappears automatically.

Please check the example from the IAPT homepage:
http://www.iapt-taxon.org/icbn/main.htm     §22
Ex. 1. The subgenus which includes the type of the name Rhododendron L. is to be named Rhododendron L. subg. Rhododendron.

**Comment by mdoering** [comment]
GBIF follows the standard tononly cite the  authorships of the terminal epithet. As there is none for autonyms we do not show any. 

It is common to show the species authorship though - for botanical names at least - like Mobot does. That should follow the species eipthet though and not the terminal one. 

I don't see a reason to change that practice in GBIF. Unless the zoological code requires sth different in which case we might need to render names differently. @deepreef any recommendation here?

**Comment by deepreef** [comment]
Can you give an example for the botanical formatting of autonyms?  In zoology, these are called nominotypical names, and the authoriship applies equally to both species and subspecies names.  It is usually shown after the terminal epithet.

**Comment by mdoering** [comment]
As far as I know in botany it follows the "protonym" epithet that actually defined the name:
For infraspecific autonyms
`Lobelia spicata Lam. var. spicata`
http://www.iapt-taxon.org/nomen/main.php?page=art26&emph=autonym

Or for subgeneric names:
http://www.iapt-taxon.org/nomen/main.php?page=art22&emph=autonym
`Rhododendron L. subg. Rhododendron`

> Publication of Tibetoseris sect. Simulatrices Sennikov (in Komarovia 5: 91. 2008) automatically established the autonym Tibetoseris Sennikov sect. Tibetoseris

**Comment by mdoering** [comment]
@deepreef I would love to treat the formatting the same for both codes, otherwise we create a new dependency that causes lots of problems.

Would a formatting like the botanic code requires upset zoologist or even contradict the code?

**Comment by deepreef** [comment]
@mdoering It works a bit differently in Zoology.  Basically, whenever anyone establishes a species-group name (either as a species or as a subspecies), the name is implied to have been made available at all ranks within the species group.  Thus, the establishment of Aus bus L. automatically implies the existence of Aus bus bus L., even if no one ever treats it at the subspecies level.  Likewise, the establishment of Aus bus dus Pyle automatically makes Aus dus Pyle available at the species level, even if no one treats it as such.

But in any case, the formatting always has the authorship after the terminal epithet.  Both epithets are treated as the same Protonym, but at different ranks. I think a simple rule could be that when a name array contains two identical consecutive protonyms in sequence (autonym/nominotypical name), then authorship is inserted after the first consecutive protonym for botanical names, and the authorship is inserted after the last consecutive protonym for zoological names.  I don't know how they do it in bacteria, but I would guess the same as botany.

**Comment by mdoering** [comment]
Damn, I feared so. Thanks for clarifying.

Does that also mean when the first subspecies for a given zoological species is published as a new protonym, the nominotypical name at the same rank is not necessarily required to exist in a consistent taxonomy? E.g. if Aus bus cus is published as the first subspecies, does Aus bus bus need to exist in a decent taxonomy? It is available, but could someone accept just Aus bus  and aus bus cus? Not sure if I explained myself well enough, its a taxonomic question.
